src/basis_factorization/ForrestTomlinFactorization.cpp

```diff
@@ -13,15 +13,18 @@
 
  **/
 
+#include "ForrestTomlinFactorization.h"
+
 #include "BasisFactorizationError.h"
 #include "EtaMatrix.h"
 #include "FloatUtils.h"
-#include "ForrestTomlinFactorization.h"
```

Comment:
This one is a bit strange. Why did it single out this include?

Reply:
I think it's trying to single out the header file of this particular cpp file. Let me try to turn this off.

Reply:
Not turning this off per off-line discussion.